src/pages/StudentDashboardPage/AssignmentSubmission/AssignmentSubmission.js

```diff
@@ -110,25 +157,74 @@ function AssignmentSubmission() {
+          <ControlPointOutlinedIcon onClick={handleOpen} />
+          <Modal
+            aria-labelledby="transition-modal-title"
+            className={classes.modal}
+            open={open}
+            onClose={handleClose}
+            closeAfterTransition
+            BackdropComponent={Backdrop}
+            BackdropProps={{
+              timeout: 500,
+            }}
+          >
+            <Fade in={open}>
+              <div className={classes.paper}>
+                <Typography style={{ textAlign: "center" }}>Upload Assignment</Typography>
+                <Divider />
+                <br />
+                <div style={{ textAlign: "center" }}>
+                  <Button
+                    variant="contained"
+                    color="default"
+                    component="span"
+                    startIcon={<AddIcon />}
+                    onClick={onChange}
+                  >
+                    Add
+                  </Button>
+                </div>
+              </div>
+            </Fade>
+          </Modal>
         </IconButton>
       </Box>
       <Box>
-        {[1, 2, 3, 4].map((items, index) => (
-          <List key={index} component="nav" className={classes.list} aria-label="contacts">
-            <ListItem>
-              <ListItemIcon>
-                <Assignment />
-              </ListItemIcon>
-              <ListItemText primary="Assignment1.pdf" />
-              <ListItemIcon>
-                <IconButton>
-                  <Delete />
-                </IconButton>
-              </ListItemIcon>
-            </ListItem>
-          </List>
-        ))}
+        {files
+          ? files.map((file, index) => (
+              <List key={index} component="nav" className={classes.list} aria-label="contacts">
+                <Link href={file.source} color="inherit" target="_blank" underline="none">
+                  <ListItem>
+                    <ListItemIcon>
+                      <Assignment />
+                    </ListItemIcon>
+                    <ListItemText primary={file.name} />
+                    <span> Size: {file.size} bytes </span>
+                    <ListItemIcon>
+                      <IconButton>
+                        <Delete />
+                      </IconButton>
+                    </ListItemIcon>
+                  </ListItem>
+                </Link>
+              </List>
+            ))
+          : [1, 2, 3].map((file, index) => (
+              <List key={index} component="nav" className={classes.list} aria-label="contacts">
+                <ListItem>
+                  <ListItemIcon>
+                    <Assignment />
+                  </ListItemIcon>
+                  <ListItemText primary="Assignments.pdf" />
+                  <ListItemIcon>
+                    <IconButton>
+                      <Delete />
+                    </IconButton>
+                  </ListItemIcon>
+                </ListItem>
+              </List>
+            ))}
```

Comment:
Hey @poojadas07  can we remove this section instead we can show a simple empty state UI component because initially `files`  array will be empty right!

Reply:
@poojadas07  can we implement an uploading state of the file like showing a percentage of the file uploaded for visual feedback to the user that would be a great user experience :)

Reply:
Hey, @poojadas07  as @Abhishek-kumar09 approve the above changes you can make a different  PR for this if you agree with the thoughts :)

Reply:
> Hey @poojadas07 can we remove this section instead we can show a simple empty state UI component because initially `files` array will be empty right!

I think it looks good ....It's like when we upload the file it will show the new file otherwise we can show the default files.

Reply:
> @poojadas07 can we implement an uploading state of the file like showing a percentage of the file uploaded for visual feedback to the user that would be a great user experience :)

I will look for this...